packages/react-testing/src/root.tsx

```diff
@@ -106,6 +106,8 @@ export class Root<Props> implements Node<Props> {
     this.acting = true;
 
     const afterResolve = () => {
+      // Flush any remaining act before updating the wrapper
+      act(() => {});
```

Comment:
You love to see it. 

Two things jump to mind:
- Can we add a test for the type of thing this is fixing so nobody deletes this despite the comment? 
- Let's use an explicit `noop` function rather than an inline one for added clarity
-